website/src/mode.tsx

```diff
@@ -225,3 +236,43 @@ const TagSetOption = (p: { radio: RadioStateReturn; level: string }) => {
     </Tooltip>
   )
 }
+
+export const PrefPanel = () => {
+  const preferences = useContext(preferencesContext)
+  return (
+    <div>
+      <h3>Display Mode:&ensp; </h3>
+      {<ExperiencePicker onSelect={preferences.expLevelUpdate} />}
+      <p>{levelNameMapping[preferences.expLevel].details}</p>
+      <h3>Romanization Method:&ensp; </h3>
+      {<PhoneticsPicker onSelect={preferences.phonRepUpdate} />}
+      <p>{phoneticRepresentationMapping[preferences.phonRep].details}</p>
+    </div>
+  )
+}
+
+export const HeaderPrefDrawer = () => {
+  const dialog = useDialogState({ animated: true })
+
+  return (
+    <div aria-label="shell" className={css.prefButton.shell}>
+      <DialogDisclosure
+        {...dialog}
+        aria-label="Open Preferences Drawer"
+        className={css.prefButton.button}
+      >
+        <MdSettings size={32} />
+      </DialogDisclosure>
+      <DialogBackdrop {...dialog} className={css.prefBG}>
+        <Dialog
+          {...dialog}
+          as="nav"
```

Comment:
This isn't a navigation element so we shouldn't use `nav`. Either `div` or perhaps `aside` should work here.